_issues/hillary_lgbt.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Doesn't she just pay lip service to the LGBT community?"
```

Comment:
Also I'm not sure if there's a shorter title we could use... I'll just leave that out there